inspirehep/modules/references/processors.py

```diff
@@ -90,11 +90,10 @@ def _is_arxiv(obj):
     """Small checker for arXiv report numbers."""
     # Workaround until this issue:
     # https://github.com/inveniosoftware/idutils/issues/14 is solved.
-
     arxiv_test = obj.split()
     if not arxiv_test:
         return False
-    return idutils.is_arxiv(arxiv_test[0])
+    return idutils.is_arxiv(arxiv_test[0]) or False
```

Comment:
:+1: I would say this is a bug of `idutils` then. Maybe you could open an issue/PR there?
Anyway, it is probably more clear to wrap return value in `bool(...)` instead of adding `or False` (with a comment to explain the reason, as it is not obvious).